src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
```suggestion
public class Lottery  {
```

Не думаю, что класс Lottery должен екстендить ColorSupplier